source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -106,7 +106,7 @@ namespace PlayFab
             }
 
             // pipeline failed to intake the event, create a response
-            const auto& playFabEmitRequest = std::dynamic_pointer_cast<const PlayFabEmitEventRequest>(request);
+            std::shared_ptr<const PlayFabEmitEventRequest> playFabEmitRequest = std::dynamic_pointer_cast<const PlayFabEmitEventRequest>(request);
             auto playFabEmitEventResponse = std::shared_ptr<PlayFabEmitEventResponse>(new PlayFabEmitEventResponse());
```

Comment:
>td::shared_ptr<PlayFabEmitEventResponse>(new [](start = 45, length = 44)

nit: total tangent so feel free to "won't fix" but you could simplify this line as `std::make_shared<PlayFabEmitEventResponse>()`